versioned_docs/version-1.0.0/provider/daemon/getting_started/run_docker.md

```diff
@@ -0,0 +1,100 @@
+    depends_on:
+      - oceand
+    environment:
+      - TDEX_WALLET_ADDR=oceand:18000
+      - TDEX_LOG_LEVEL=5
+      - TDEX_NO_OPERATOR_TLS=true
+      - TDEX_CONNECT_PROTO=http
+    ports:
+      - "9000:9000"
+      - "9945:9945"
+    volumes:
+      - volumes/tdexd:/home/tdex/.tdex-daemon
+      - volumes/tdex-cli:/home/tdex/.tdex-operator
+  oceand-db:
+    container_name: oceand-db
+    image: postgres
+    restart: unless-stopped
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=secret
+      - POSTGRES_DB=oceand-db
+    ports:
+      - "5432:5432"
+```
+
+This compose file lets you serve your daemon on the testnet network but it can be used as good starting point for your mainnet provider as well.
+
+3 docker services are defined here:
+* `oceand`, the Ocean wallet service, required by the daemon to craft its transactions (deposits, withdrawals, trades)
+* `tdexd`, the tdex daemon that lets you setup and open markets, deposit or withdraw funds and many other operations.
+* `oceand-db` a postgres db for the ocean wallet, as a more consistent data storage rather than relying on the filesystem. If you want to use the latter instead, you can drop the `OCEAN_DB_HOST` env var and add a new one `OCEAN_DB_TYPE=badger`.
+
+Place this file in your preferred folder and name it `docker-compose.yml`, take care of editing the volumes folders mounted for `oceand` and `tdexd` if you need. With the configuration above, instead, you should take care of creating the folders `volumes/oceand`, `volumes/ocean-cli`, `volumes/tdexd` and `volumes/tdex-cli` in the same directory where you placed the compose file. This should be the structure of your main directory:
+
+```
+|__ docker-compose.yml
+|__ volumes/
+   |__ oceand/
+   |__ ocean-cli/
+   |__ tdexd/
+   |__ tdex-cli /
+```
+
+You're now ready to start up the services. First of all, let's start the ocean services:
+
+```bash
+$ docker-compose up -d oceand-db oceand
+```
+
+You can inspect the status of the service by running `docker ps` - don't be afraid if oceand gives errors at this stage, it can happen depending on how fast the db startup phase is. Once both services are up and running you can start up your provider service:
+
+```bash
+$ docker-compose up -d tdexd
+```
+
+That's it! You just started your provider served as docker container. What's next? You can get some insights about [the daemon's configuration](configure_daemon.md), or you can take a look at how to [configure the CLI](configure_cli.md) to setup your markets, deposit and withdraw funds and even more cool stuff.
+
+If you are asking where this CLI is, it comes with your daemon service and you can use it by just creating the following alias:
+
+```bash
+alias tdex="docker exec tdexd tdex"
+```
+
+:::tip
+If you struggle starting up the wallet or daemon services and you see some _Permission denied_ error from their logs, try to change the permissions of your volumes folder with `chmod -R 777 volumes`.
```

Comment:
this is worth mentioning to check the user permissions first

``
docker run -it --user 0:0 ...
```

from https://github.com/iotaledger/goshimmer/issues/623#issuecomment-653640862

Reply:
owh! you're right, better to change user rather than chmodding but from my my personal experience using `0` (root) causes problems since you then have to use `sudo` to surf the volumes.

Our docker services by default use the user `1000` which is typically `ubuntu` on Linux/Debian, if this doesn't work, then it should be enough tu use `1001` but in general the way to go is finding the id of the `ubuntu` user with `less /etc/passwd`.

We can add this as a tip